containers/podman-desktop
# feat: Add start on boot and start container engine on launch

### What does this PR do?

- Automatically starts Podman Desktop when logging/starting a session on Windows and macOS
- Automatically starts the first podman machine when Podman Desktop is starting. (No machine = no automatic start)

It supports Windows and macOS

note: on macOS, you need to have Podman Desktop in Applications or user Applications folder.
It does not work for development mode as well (macOS or Windows) (when it's started using `yarn start`)

### Screenshot/screencast of this PR

https://user-images.githubusercontent.com/436777/193537580-e453739d-fc89-4803-a1f1-70c91e5c260d.mp4



### What issues does this PR fix or reference?
fixes https://github.com/containers/podman-desktop/issues/225


### How to test this PR?

On Windows: try to see if Podman Desktop is starting automatically on startup (and machine is starting as well)
On macOS: try to see if Podman Desktop is starting automatically on startup (and machine is starting as well)

Turn off/turn on the preferences in the settings page



Change-Id: I6b0f0bc31f3427a41b28dbb87ac620fa1ecbc7b9
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
